modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/IdPSuccessTest.java

```diff
@@ -128,12 +128,13 @@ public void testListMetaFederatedAuthenticators() throws Exception {
                         ("/t/" + context.getContextTenant().getDomain() +
                                 "/api/server/v1/identity-providers/meta/federated-authenticators" +
                                 "/TWljcm9zb2Z0V2luZG93c0xpdmVBdXRoZW50aWNhdG9y"))
-                .body("find{ it.authenticatorId == 'UGFzc2l2ZVNUU0F1dGhlbnRpY2F0b3I' }.name", equalTo
-                        ("PassiveSTSAuthenticator"))
-                .body("find{ it.authenticatorId == 'UGFzc2l2ZVNUU0F1dGhlbnRpY2F0b3I' }.self", equalTo("/t/" +
-                        context.getContextTenant().getDomain() +
-                        "/api/server/v1/identity-providers/meta/federated-authenticators" +
-                        "/UGFzc2l2ZVNUU0F1dGhlbnRpY2F0b3I"))
+                // Disabling since passiveSTS removed from product
```

Comment:
Shall we remove the commented codes as well, since there are no longer needed for this distribution?